lib/graphql/index.ts

```diff
@@ -0,0 +1,19 @@
+import { User } from "./user";
+import { Account } from "./account";
+import { Transfer } from "./transfer";
+import { Transaction } from "./transaction";
+import { GraphQLClient } from "./client";
+
+export type Models = {
+  transaction: Transaction;
+  transfer: Transfer;
+  account: Account;
+  user: User;
+};
+
```

Comment:
Can we just use type interference here and remove this type?
<img width="628" alt="Bildschirmfoto 2019-11-01 um 13 19 04" src="https://user-images.githubusercontent.com/6780316/68024627-ec438a80-fcaa-11e9-9f25-15bd9dec55da.png">

then in `client.ts` either use `  public models: ReturnType<typeof getModels>;` or even just via constructor assignment like this (which btw could simplify unit testing as well) :

```typescript
export class Client {
  constructor(
    opts: ClientOpts,
    baseUrl = opts.baseUrl || KONTIST_API_BASE_URL,
    public auth = new Auth(baseUrl, opts),
    public graphQL = new GraphQLClient(`${baseUrl}/api/graphql`, auth),
    public models = getModels(graphQL)
    ) { }
}
```